var/spack/repos/builtin/packages/brayns/package.py

```diff
@@ -68,3 +69,9 @@ def cmake_args(self):
     def check(self):
         with working_dir(self.build_directory):
             ninja('tests')
+
+    def build(self, spec, prefix):
+        chdir('spack-build')
+        if '+optix' in self.spec:
+            ninja('braynsOptixEngine')
```

Comment:
you could also 'ninja -C spack-build braynsOptixEngine', or `with working_dir(self.build_directory)` like above in check()

Reply:
I think the latter would be best.

Reply:
done